doc/dftb+/manual/dftbp.tex

```diff
@@ -3048,14 +3048,20 @@ \subsubsection{Generalized Born Model}
     \is{FromConstants\cb} requires the dielectric constant as real for \kw{Epsilon},
     the molecular mass in \kw{MolecularMass} \modif{\modtype{mass}} and the
     solvent density in \kw{Density} \modif{\modtype{massdensity}}.
+    \kw{MolecularMass} and \kw{Density} only affect the calculation if
+    ``reference'' is chosen as state of the solvent.
 
-  \item[\is{Shift}] \modif{\modtype{energy}} Shift for free energy calculations.
+  \item[\is{FreeEnergyShift}] \modif{\modtype{energy}} Shift for free energy
+    calculations.
 
-  %\item[\is{Temperature}] Temperature for free energy calculations. Default is
-    %ambient temperature: 298.15 K.
+  \item[\is{Temperature}] Temperature for free energy calculations. Default is
+    ambient temperature: 298.15 K.
+    Only affects the calculation for if ``reference'' or ``mol1bar'' is chosen
```

Comment:
Might want to briefly explain what mol1bar corresponds to

Reply:
The safest choice in my opinion is always “gsolv” as state, as you can more easily arrive from there at the really wanted state. Of course you than have to keep in mind that you might have to add the volume work yourself afterwards.

Reply:
Just added a short description on the available states.